tabs/pid_tuning.js

```diff
@@ -701,11 +718,8 @@ TABS.pid_tuning.initRatesPreview = function () {
     this.keepRendering = true;
     this.model = new Model($('.rates_preview'), $('.rates_preview canvas'));
 
-    var scale = d3.scale.linear().domain([900, 2100]);
-
-    this.rollScale = scale.range([Math.PI * 2, -Math.PI * 2]);
-    this.pitchScale = scale.range([Math.PI * 2, -Math.PI * 2]);
-    this.yawScale = scale.range([Math.PI * 2, -Math.PI * 2]);
+    this.$superExpo = $('.pid_tuning input[name="show_superexpo_rates"]');
```

Comment:
Well I've put `$superExpo` on the object just to avoid scope wrapping, which I think is over-abused in this project, which makes is really messy.

Anyways, I've refactored it a bit — #93, should be cleaner now and should avoid problems you've mentioned.